core/storage/job/gae_models.py

```diff
@@ -86,6 +93,17 @@ def is_cancelable(self):
 
     @classmethod
     def get_recent_jobs(cls, limit, recency_msec):
+        """Gets at most limit jobs with respect to a time after recency_msec.
+
+        Args:
+            limit: int. A limit on the number of jobs to return.
+            recency_msec: int. A time in milliseconds. After this time a
+                job is considered recent.
+
+        Returns:
+            list(Jobs) or None. A list of at most limit jobs
```

Comment:
This involves a bit of "code chasing". Search the codebase to find which methods use this function, and search the codebase to find which methods use that method, etc., until you find something at the controller layer that you can trigger (at, say, the admin page). Then start up a local dev server, navigate to that page, and perform the action that leads to this function being triggered. You should see the server logs in the terminal.

I suggest having a go, but if you're stuck, try and formulate specifically why you're stuck, and feel free to reply to this comment and/or ask on Gitter. We can help you from there.